src/pages/api/client_sessions/get.ts

```diff
@@ -0,0 +1,59 @@
+import { BadRequestException, NotFoundException } from "nextlove"
+import { z } from "zod"
+
+import { client_session } from "lib/zod/index.ts"
+
+import { withRouteSpec } from "lib/middleware/with-route-spec.ts"
+
+export default withRouteSpec({
+  auth: "cst_ak_pk",
+  methods: ["GET", "POST"],
+  commonParams: z.object({
+    client_session_id: z.string().optional(),
+    user_identifier_key: z.string().optional(),
+  }),
+  jsonResponse: z.object({
+    client_session,
+  }),
+} as const)(async (req, res) => {
+  const { commonParams, auth } = req
+
+  const is_client_session =
```

Comment:
is this missing throwIfNotAllowed? I think we also need to enable this endpoint in the middleware

Reply:
*whitelist this endpoint

Reply:
THIS IS THE FAKE